django/db/backends/postgresql/operations.py

```diff
@@ -271,5 +274,17 @@ def explain_query_prefix(self, format=None, **options):
             prefix += ' (%s)' % ', '.join('%s %s' % i for i in extra.items())
         return prefix
 
-    def ignore_conflicts_suffix_sql(self, ignore_conflicts=None):
-        return 'ON CONFLICT DO NOTHING' if ignore_conflicts else super().ignore_conflicts_suffix_sql(ignore_conflicts)
+    def conflicts_suffix_sql(
+        self, opts, fields, on_conflicts=ON_CONFLICTS_NONE, update_fields=[], unique_fields=[]
+    ):
+        result = ''
+        if on_conflicts == ON_CONFLICTS_IGNORE:
+            result = 'ON CONFLICT DO NOTHING'
```

Comment:
This method implementation could be simplified by doing:

```python
if on_conflicts == ON_CONFLICTS_IGNORE:
    return 'ON CONFLICT DO NOTHING'
if on_conflicts == ON_CONFLICTS_UPDATE:
    ...
    return result
...
```

This would also let you eliminate the `if-else` below.